community/libmupdf/build

```diff
@@ -1,5 +1,8 @@
 #!/bin/sh -e
 
+export CFLAGS="$CFLAGS -fPIC"
+export CXXFLAGS="$CXXFLAGS -fPIC"
+
```

Comment:
I'd rather we didn't set CFLAGS like this unless it makes sense (coreutils as static for example). This can be done through your CFLAGS or for finer control, through `KISS_HOOK`.

Reply:
Does this fix a compile bug with our normal GCC or your modified one? If the latter, my previous message applies.

Reply:
I'm running our normal GCC. These C{,XX}FLAGS are [necessary](https://bugzilla.redhat.com/show_bug.cgi?id=1109589) regardless, I assume due to the fact that `libmupdf` is static only and `zathura-pdf-mupdf` is a shared object, and so requires all PIC.

Reply:
I can reproduce. Must both CFLAGS and CXXFLAGS be set?

Reply:
Turns out just CFLAGS.